### PR TITLE
[easee] Update README.md - added status values for "state#chargerOpMode"

### PR DESCRIPTION
**Pull request by WoKe77**
added status values for "state#chargerOpMode"

### CODE REVIEW

File: bundles/org.openhab.binding.easee/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The settings that start with "dynamic" can be changed frequently, the others are
...
 | state#cableLocked                           | Switch                   | no       |                                                      |                                                                                                                                                              |
-| state#chargerOpMode                         | Number                   | no       |                                                      |                                                                                                                                                              |
+| state#chargerOpMode                         | Number                   | no       | Offline=0, Disconnected=1, AwaitingStart=2, Charging=3, Completed=4, Error=5, ReadyToCharge=6                                                                                                                   |                                                                                                                                                              |
```
Comment (lolodomo) [inline bundles/org.openhab.binding.easee/README.md:63]: Maybe rather something like: 0 for offline, 1 for disconnected, ... ?

**Review by alexf2015 — CHANGES_REQUESTED** [review]
as commented please change bthe syntax according to my comment. Apart from that I am fine with your changes

File: bundles/org.openhab.binding.easee/README.md (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.easee/README.md:63]: Would also be ok for me using 0=Offline...

Comment (lolodomo) [reply bundles/org.openhab.binding.easee/README.md]: As there was no feedback, I fixed myself.

**Comment by lolodomo** [comment]
@alexf2015 : please approve my change.

**Review by jlaur — APPROVED** [review]
Thanks all.

**Pull request merged**